Dockerfile

```
@@ -0,0 +1,19 @@
+FROM golang:1.16.4 as base
+
+ENV GO111MODULE=on
+WORKDIR /app
+
+COPY go.mod .
+COPY go.sum .
+
+RUN go mod download
+
+COPY . .
+
+FROM base AS builder
+RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build .
```

```suggestion
RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build . -ldflags='-s -w -extldflags "-static"'
```

Stripping reduces binary size, fully static linking makes the binary more portable